roles/oradb-manage-db/templates/dotprofile-db.j2

```diff
@@ -46,12 +25,4 @@ get_sid=$(ps -ef | grep "ora_pmon_$ORACLE_DBNAME" |grep -v grep | sed 's/^.*pmon
         export RMANSCRIPTDIR={{ rman_script_dir | default(dbh.rman_script_dir | default(oracle_base + '/rman/script')) }}
 {% endif %}
 
-export PATH=$ORACLE_HOME/bin:$ORACLE_HOME/OPatch:$PATH:$SQLPATH:$GG_HOME/:$ORACLE_BASE/bin:$ORACLE_HOME/suptools/oratop
-
-# Set Up Aliases:
-{% if shell_aliases is defined %}
-{% for alias in shell_aliases %}
-        alias {{ alias }}
-{% endfor %}
-{% endif %}
-
+export PATH=$ORACLE_BASE/bin:$ORACLE_HOME/suptools/oratop
```

Comment:
old "PATH" variable gets lost here

Reply:
old .profile_<SID> is replaced by ocenv and not used anymore.